db.js

```diff
@@ -32,6 +32,20 @@ module.exports.CreateDB = function (parent, func) {
     var expirePowerEventsSeconds = (60 * 60 * 24 * 10);         // By default, expire power events after 10 days (864000). (Seconds * Minutes * Hours * Days)
     var expireServerStatsSeconds = (60 * 60 * 24 * 30);         // By default, expire power events after 30 days (2592000). (Seconds * Minutes * Hours * Days)
     const common = require('./common.js');
+    const path = require('node:path');
+    const fs = require('node:fs');
```

Comment:
dont use this require format as its a new format!
altho we set meshcentral to a min of node 16,
the is still a tiny/tiny/tiny percentage of people out there in the world
that run a custom version which uses node 10 !?
can you change these to `path` and `fs` plz

Reply:
Done. I did all this with min v16 in mind. Did some checks with a v10 node, everything worked. Also did a mariadb check now, also works.

Reply:
thank you! i just wanted all the code the same as the others,
then in the future IF we have to move to `node:xyz` we can just just a find and replace haha